app/src/main/java/ch/epfl/sweng/studdybuddy/util/CoreFactory.java

```diff
@@ -0,0 +1,48 @@
+package ch.epfl.sweng.studdybuddy.util;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import ch.epfl.sweng.studdybuddy.core.Course;
+import ch.epfl.sweng.studdybuddy.core.Group;
+import ch.epfl.sweng.studdybuddy.core.Pair;
+import ch.epfl.sweng.studdybuddy.core.User;
+
+public final class CoreFactory {
+    public static List<Pair> userGroup1() {
+        return Arrays.asList(new Pair("456", "123"), new Pair("789", "123"), new Pair("ghi", "abc"));
+    }
+
+    public static Group blankGroupWId(String id) {
+        return new Group(1, new Course(""), "", id, UUID.randomUUID().toString());
+    }
+
+    public static Group blankGroupWithCourse(String course) {
+        return new Group(1, new Course(course), "", "", "");
+    }
+
+    public static Group groupOf(int parti) {
+        return new Group(parti, new Course("test"), "fr", UUID.randomUUID().toString());
+    }
+
+
+    public static User johnDoe(String id) {
```

Comment:
I take it that all these are for the tests. Maybe we should place them elsewhere in the future

Reply:
Yes maybe in the test package, but I don't think it is inconvenient to let it as is for now